atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -247,14 +247,15 @@ public boolean hasNext() {
                         runningSize += sizingFunction.apply(firstEntry);
                         entries.add(firstEntry);
                         if (runningSize > maximumBytesPerPartition && log.isWarnEnabled()) {
-                            String message = "Encountered an entry of approximate size {} bytes, "
+                            final String message = "Encountered an entry of approximate size {} bytes, "
```

Comment:
In fact, we should block this PR on #1918, because all of these fixes depend on the kind of static check we decide to use.